facebook/react-native
# fix: dark mode in SnapshotExample in RNTester

## Summary:

This PR fixes issue in `RNTester` causing labels and image background to not be visible in dark mode in `SnapshotExample`

It also fixes issue with description in `Header` not being visible in other components examples when using dark mode

Before & After
<img width="505" alt="image" src="https://github.com/facebook/react-native/assets/56474758/ce87df69-4b79-48a0-b9be-4a7335329b78">


## Changelog:

[INTERNAL] [FIXED] - Fix dark mode in SnapshotExample in RNTester

## Test Plan:

1. Launch `RNTester` with dark mode enabled
2. Open `Snapshot / Screenshot` example
3. All labels should be visible, image background should have white color
